app/build.gradle.kts

```diff
@@ -1,23 +1,25 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.compose.compose
 
 group = "me.john_"
 version = "1.0-SNAPSHOT"
 
 plugins {
-    kotlin("jvm") version "1.4.10"
+    kotlin("jvm") version "1.4.21-2"
+    id("org.jetbrains.compose") version "0.3.0-build146"
     java
 }
 
 repositories {
+    flatDir {dirs = setOf(file("libs")) } // have to move it into the a library folder
+    jcenter()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
     mavenLocal()
     mavenCentral()
-    jcenter()
-    flatDir {dirs = setOf(file("libs")) } // have to move it into the a library folder
-
 }
 dependencies {
     implementation( "com.mcts:lib:1.0-SNAPSHOT" )
     implementation( "de.magoeke.kotlin:connectfour:1.0-SNAPSHOT" )
+    implementation(compose.desktop.currentOs)
```

Comment:
@JunTaoLuo is detection of currentOs required? Will this be platform independent?

Reply:
I think it's because the jetpack compose GUI framework installs a slightly different package on different OS's. I doubt that it'll be a compatibility issue since the framework should be cross platform.